airflow/README.md

```diff
@@ -16,22 +17,46 @@ Download METS file for binding 379973 and all ALTO files listed in the downloade
 
     ```echo -e "AIRFLOW_UID=$(id -u)" > .env```
 
-3. Initialize Airflow in docker:
+3. In order to run the Puhti dag, also add the absolute path to your `.ssh` folder to the `.env` file in the following manner:
+
+    ```SSH_PATH=/your/path/.ssh```
```

Comment:
The previous step uses echo to pipe content into the file, so it would be neat if this did too. So something like `echo "SSH_PATH=/your/path/.ssh" >> .env`

Or uniformly asking to manually edit the file is fine too.